src/main/java/org/broadinstitute/hellbender/utils/test/BaseTest.java

```diff
@@ -46,6 +46,9 @@
 
     public static final String hg19_chr1_1M_Reference = publicTestDir + "Homo_sapiens_assembly19_chr1_1M.fasta";
     public static final String hg19_chr1_1M_dbSNP = publicTestDir + "Homo_sapiens_assembly19.dbsnp135.chr1_1M.exome_intervals.vcf";
```

Comment:
could you put a line break here so that it's obvious which file is being referred to even without saying "the following file"